.github/workflows/ci-test-ginkgo.yml

```diff
@@ -54,6 +54,12 @@ jobs:
         run: |
           make docker-build
 
+      - name: deploy pre existing pod 
+        run: |
+          kubectl apply -f ./tests/syscalls/manifests/pre-run-pod.yaml
+          sleep 60
+          kubectl get pods -A
```

Comment:
@Shreyas220 I see that we've included these only in the CI. What if someone runs ginkgo locally?